iOCNotes/fr.lproj/Localizable.strings

```diff
@@ -1,147 +1,147 @@
+"Connection failure" = "Échec de connexion";
 
 /* Alert view title */
-"Crash Detected" = "Crash Detected";
+"Crash Detected" = "Crash détecté";
 
 /* A menu action */
-"Delete Note" = "Delete Note";
+"Delete Note" = "Supprimer la note";
 
 /* The title of an error message */
-"Error Adding Note" = "Error Adding Note";
+"Error Adding Note" = "Erreur lors de l'ajout de la note";
 
 /* The title of an error message */
-"Error Deleting Note" = "Error Deleting Note";
+"Error Deleting Note" = "Erreur lors de la suppression de la note";
 
 /* The title of an error message */
-"Error Getting Note" = "Error Getting Note";
+"Error Getting Note" = "Erreur lors de l'obtention de la note";
 
 /* The title of an error message */
-"Error Syncing Notes" = "Error Syncing Notes";
+"Error Syncing Notes" = "Erreur de synchronisation des notes";
 
 /* The title of an error message */
-"Error Updating Note" = "Error Updating Note";
+"Error Updating Note" = "Erreur lors de la mise à jour de la note";
 
 /* The title of an error message */
-"Error Updating Notes" = "Error Updating Notes";
+"Error Updating Notes" = "Erreur lors de la mise à jour des notes";
 
 /* An error message */
-"Failed to connect to a server. Check your settings." = "Failed to connect to a server. Check your settings.";
+"Failed to connect to a server. Check your settings." = "Échec de la connexion à un serveur. Vérifiez vos paramètres.";
 
 /* A menu option for plain sharing in html format */
 "HTML" = "HTML";
 
 /* A status label indicating that the user is logged in */
-"Logged In" = "Logged In";
+"Logged In" = "Connecté";
 
 /* A menu option for sharing in markdown format */
 "Markdown" = "Markdown";
 
 /* The title of a new note */
-"New note" = "New note";
+"New note" = "Nouvelle note";
 
 /* No comment provided by engineer. */
-"No thanks" = "No thanks";
+"No thanks" = "Non merci";
 
 /* A status label indicating that the user is not logged in */
-"Not Logged In" = "Not Logged In";
+"Not Logged In" = "Pas connecté";
 
 /* An error message */
-"Notes could not be found on your server. Make sure it is installed and enabled" = "Notes could not be found on your server. Make sure it is installed and enabled";
+"Notes could not be found on your server. Make sure it is installed and enabled" = "Notes sont introuvables sur votre serveur. Assurez-vous qu'il est installé et activé";
 
 /* An error message title */
-"Notes not found" = "Notes not found";
+"Notes not found" = "Notes non trouvées";
 
 /* Title for Open In activity */
-"Open In..." = "Open In...";
+"Open In..." = "Ouvrir dans...";
 
 /* A menu option for sharing in plain text format */
-"Plain Text" = "Plain Text";
+"Plain Text" = "Texte brut";
 
 /* A message
    A message to check network connection */
-"Please check network connection and login." = "Please check network connection and login.";
+"Please check network connection and login." = "Veuillez vérifier la connexion réseau et la connexion.";
```

Comment:
> Veuillez vérifier la connexion réseau et re-connectez vous.

not sure if you mean one must login again or check the login value. I assume one must login again in the translation